src/chess/UI/chessgame.js

```diff
@@ -326,7 +345,7 @@ class ChessGame extends React.Component {
                 const currentBoard = currentGame.getBoard()
                 const finalPosition = currentBoard[to_coords[0]][to_coords[1]].getCanvasCoord()
                 const selectedId = currentGame.chessBoard[from_coords[0]][from_coords[1]].pieceOnThisSquare.id
-                this.movePiece(selectedId, finalPosition, currentGame, true)
+                //this.movePiece(selectedId, finalPosition, currentGame, true)
```

Comment:
Why is the movePiece line commented out?

Reply:
Redundant comment, I've removed it.